instrumentation/pg/README.md

```diff
@@ -42,6 +42,12 @@ OpenTelemetry::SDK.configure do |c|
     # You may optionally set a value for 'peer.service', which
     # will be included on all spans from this instrumentation:
     peer_service: 'postgres:readonly',
+
+    # By default, this instrumentation includes the executed SQL as the `db.statement`
+    # semantic attribute (possibly sanitized, according to the value of `enable_sql_obfuscation`).
+    # Optionally, you may disable the inclusion of this attribute entirely by
+    # setting this option to false.
+    enable_db_statement_attribute: true,
```

Comment:
Perhaps just:
```suggestion
    enable_statement_attribute: true,
```

In #737 I suggested normalizing the obfuscation option across instrumentation as `enable_statement_obfuscation`. This rename would line up with that. I'm not sure we need the `db_` name spacing on the option.

Reply:
👍 That makes sense to me, and it's changed in c996c4e2f2403bd0d3843086059cb52950b57f26.

Re: `enable_statement_obfuscation` specifically, I did not change that in this PR, although I think you have a good point there. Should we rename that in this PR, or do it together with mysql in another (and, do we care about maintaining compatibility for the old name at all)?

Reply:
Let's do it in a separate PR. I don't think we need to maintain compatibility pre-1.0, but we should call it out prominently in the release notes.